src/bundles/site-logo.js

```diff
@@ -3,26 +3,26 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import urls from '@educandu/educandu/utils/urls.js';
 
-function Logo({ readonly, size }) {
+function SiteLogo({ readonly, size }) {
   const classes = classNames({
-    'Logo': true,
-    'Logo--readonly': readonly,
-    'Logo--big': size === 'big'
+    'SiteLogo': true,
+    'SiteLogo--readonly': readonly,
+    'SiteLogo--normal': size === 'normal'
```

Comment:
Isn't it a little bit weird, that `normal` is an override of `small`? I found it more logical to have `big` as an override of `normal`.

Reply:
Actually this is a leftover from the old `site-logo`. I made it better now, please check.